lib/rmt/cli/products.rb

```diff
@@ -0,0 +1,62 @@
+# rubocop:disable Rails/Output
+
+require 'rmt/cli/repos'
+
+class RMT::CLI::Products < RMT::CLI::Base
+
+  desc 'list', 'List all products'
+  option :release_stage, aliases: '-r', type: :string, desc: 'beta, released'
+  def list
+    attributes = %i[id name version arch product_string release_stage mirror? last_mirrored_at]
+    headings = ['ID', 'Name', 'Version', 'Architecture', 'Product string', 'Release stage', 'Mirror?', 'Last mirrored']
+
+    conditions = options[:release_stage] ? { release_stage: options[:release_stage] } : {}
+
+    rows = Product.where(conditions).map do |product|
+      attributes.map { |a| product.public_send(a) }
+    end
+
+    if rows.empty?
+      warn 'No products found in the DB. Please run "rmt-cli scc sync" to synchronize with SUSE Customer Center first.'
+    else
+      puts Terminal::Table.new headings: headings, rows: rows
+    end
+  end
+
+  desc 'enable', 'Enable mirroring of product repositories by product ID or product string'
+  def enable(target)
+    repo_id = Integer(target, 10) rescue nil
+    if repo_id
+      change_product_mirroring_by_id(repo_id, true)
+    else
+      identifier, version, arch = target.split('/')
+      RMT::CLI::Repos.change_product_mirroring(true, identifier, version, arch)
+    end
+  end
```

Comment:
The enable and disable functions are almost identical. You could possibly simplify this.